components/View/index.jsx

```diff
@@ -84,6 +99,19 @@ class View extends Component {
     }
 
     this.props.setTop(true);
+
+    if (this.props.noScrollOnKeyboard) {
+      this.bindKeyboardChange();
+    }
+  }
+
+  /**
+   * Unbinds keyboard listener
+   */
+  componentDidUnmount() {
```

Comment:
This lifecycle hook doesn't exists. I think you mean `componentWillUnmount`

Reply:
Whoa! Changed.